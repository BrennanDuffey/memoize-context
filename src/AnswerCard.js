import React from 'react';

const AnswerCard = (props) => {
  return (
    <article>
      <h2>{props.correctAnswer}</h2>
    </article>
  )
}

export default AnswerCard;