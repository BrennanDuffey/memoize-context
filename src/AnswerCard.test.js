import React from 'react';
import AnswerCard from './AnswerCard.js';
import { shallow } from 'enzyme';

let mockAnswer = 'Undefined.'

describe('Answer components', () => {
  
  let wrapper;

  beforeEach(() => {
    wrapper = shallow (
      <AnswerCard  
        answer={mockAnswer}
      />
    );
  });

  it('Should render correctly with all the information passed', () => {
    expect(wrapper).toMatchSnapshot();
  });
});