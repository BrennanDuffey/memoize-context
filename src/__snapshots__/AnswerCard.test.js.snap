// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Answer components Should render correctly with all the information passed 1`] = `
<article>
  <h2>
    Correct Answer: 
  </h2>
</article>
`;
